utils/novnc_proxy

```diff
@@ -191,9 +200,9 @@ else
     fi
 fi
 
-echo "Starting webserver and WebSockets proxy on port ${PORT}"
-#${HERE}/websockify --web ${WEB} ${CERT:+--cert ${CERT}} ${PORT} ${VNC_DEST} &
-${WEBSOCKIFY} ${SYSLOG_ARG} ${SSLONLY} ${FILEONLY_ARG} --web ${WEB} ${CERT:+--cert ${CERT}} ${KEY:+--key ${KEY}} ${PORT} ${VNC_DEST} ${HEARTBEAT_ARG} ${IDLETIMEOUT_ARG} ${RECORD_ARG} ${TIMEOUT_ARG} ${WEBAUTH_ARG} ${AUTHPLUGIN_ARG} ${AUTHSOURCE_ARG} &
+echo "Starting webserver and WebSockets proxy on host ${HOST} port ${PORT}"
```

Comment:
```suggestion
echo "Starting webserver and WebSockets proxy on port $LISTEN"
```

Your suggestion is slightly incorrect. The default setting listens to all interfaces but the message will tell you it listens on a single interface.